bridge/src/commonMain/kotlin/ru/herobrine1st/matrix/bridge/telegram/TelegramWorker.kt

```diff
@@ -54,14 +54,32 @@ class TelegramWorker(
         actorId: TelegramActorId,
         id: ChatId
     ): RemoteRoom<ChatId> {
-        TODO("Not yet implemented")
+        val bot = getBot(actorId)
+        val chat = withContext(Dispatchers.IO) { bot.getChat(id).get() }
+
```

Comment:
```suggestion

        check(chat.type != "channel") { "Can't create room for a channel" }

```
Возможно, в будущем будет добавлена поддержка read-only комнат. Сейчас её нет